Sources/TSCUtility/IndexStore.swift

```diff
@@ -19,16 +19,20 @@ public final class IndexStore {
         public var methods: [String]
     }
 
+    /// Wrapper for `indexstore_t` so we can use implementationOnly import.
+    struct StoreWrapper {
+        let store: indexstore_t
+    }
```

Comment:
Hm, this doesn't make sense. If this is okay, then the original code should be okay as well, which makes me think they're both broken. Maybe we have a hole in implementation-only import checking for non-library-evolution libraries.

cc @xymus

Reply:
Specifically, the concern is not being able to compute the layout of the class, in case there's a public property after the implementation-only one. For library evolution libraries this is okay because they all have to be accessed indirectly anyway, but I don't think the same is true of non-library-evolution libraries.

Reply:
I see. Do you want me to file a bug?

Reply:
Yes please!

Reply:
Here: https://bugs.swift.org/browse/SR-11613